### DISCUSSION TITLE
Add local transaction history to keypad-only POS

### DISCUSSION THREAD

**Discussion by vindard** [btcpayserver/btcpayserver#5349] [Ideas / Feature Requests]
In a nutshell, it's sometimes difficult for a cashier to check if a previous invoice actually went through in instances where they might be doing lots of transaction quickly or there is some network blip or accidental page refresh.

Since POS apps already have unique order ids for each invoice, and there's a dashboard button for seeing that POS's invoices only, it may not be a heavy lift to expose this in some way to the POS end-user as well.

An example could be like how Ibex does it with a little "tx history" button in the top-right corner ([try live demo here](https://ibexpay.ibexmercado.com/bpt/6398c017-f8e3-40aa-9c22-54224ece366b))

### Example screenshot from IbexPay

![image](https://github.com/btcpayserver/btcpayserver/assets/17693119/f7ad46f8-82ef-42a6-9668-0b02dbbfd7ba)

**Comment by dennisreimann** [comment]
We are currently gathering feedback on how to improve the POS and also the refund process associated with it, so this is really helpful. @dstrukt

**Reply by pavlenex** [reply]
Thanks @vindard we received this feedback already in Riga, and I think @Kukks had a similar idea, I'd say we move this to issue to be tackled since it's already been requested in a live-life scenario? I can convert it if so.
